### DISCUSSION TITLE
Install Smilei issue

### DISCUSSION THREAD

**Discussion by lucas500126** [SmileiPIC/Smilei#752] [Q&A]
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Package python3-matplotlib is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Unable to locate package python3-h5py
E: Unable to locate package python3-ipython
E: Package 'python3-matplotlib' has no installation candidate
E: Unable to locate package libhdf5-openmpi-dev~

What is the problem?

**Comment by mccoys** [comment]
I moved this to the discussions. Please do not ask for help in the issues.

**Comment by mccoys** [comment]
Please provide more info on your system and which command you were trying
